backend/views.py

```diff
@@ -21,3 +21,13 @@ def addTask(request):
     except Exception as e:
         return JsonResponse({'fail': 'Task creation failed'})
     return JsonResponse({'success': True})
+
+@api_view(['PUT'])
+def updateTask(request):
```

Comment:
I don't know python or django, but I think this path and route handler should be called something more specific to what it actually does.

eg. the path would be something like: `/toggle-task-completed` which maps to the `toggleTaskCompleted` route hanlder